entql/internal/gen.go

```diff
@@ -74,7 +74,7 @@ func ops(t field.Type) []string {
 
 func ident(t field.Type) string {
 	switch t {
-	case field.TypeBytes:
+	case field.TypeBytes, field.TypeJSON:
```

Comment:
```suggestion
	case field.TypeBytes:
```
I'm pretty sure this one is not needed.